### PR TITLE
vacation2019/ts/networkgraph-draggable-nodes

### LINKED ISSUE #11431: Migrate `modules/networkgraph/draggable-nodes` to TypeScript

### PR DESCRIPTION
**Pull request by bre1470**
Fixed #11431 - Migrated DragNodesMixin to TypeScript.

### CODE REVIEW

File: js/modules/networkgraph/draggable-nodes.js [hunk 1]
```diff
@@ -1,37 +1,33 @@
 /* *
- * Networkgraph series
  *
- * (c) 2010-2019 Paweł Fus
+ *  Networkgraph series
```
Comment (pawelfus) [inline js/modules/networkgraph/draggable-nodes.js]: Actually, PackedBubble series also uses this module. Should move this to mixins or modules?

File: ts/modules/networkgraph/draggable-nodes.ts [hunk 2]
```diff
@@ -0,0 +1,253 @@
+/* *
+ *
+ *  Networkgraph series
```
Comment (pawelfus) [inline ts/modules/networkgraph/draggable-nodes.ts]: Actually, PackedBubble series also uses this module. Should move this to mixins or modules?

Comment (bre1470) [reply ts/modules/networkgraph/draggable-nodes.ts]: Can we delay this until after this migration phase?

Comment (pawelfus) [reply ts/modules/networkgraph/draggable-nodes.ts]: Yes, of course! I'm just thinking out loud :)

**Pull request merged**